jnosql-lite/mapping-lite-document-test/src/test/java/org/eclipse/jnosql/lite/mapping/entities/PersonCrudRepositoryTest.java

```diff
@@ -111,13 +108,6 @@ void shouldDeleteEntity() {
         verify(template, times(1)).delete(eq(Person.class),eq(person.getId()));
     }
 
-    @Test
-    void shouldDeleteAllEntities() {
```

Comment:
could you return this method?

Reply:
The PersonCrudRepository is a `CrudRepository` specialization... and `CrudRepository` doesn't have `deleteAll()` method. Should I make `PersonCrudRepository` a specialization of `NoSQLRepository`?

Reply:
Nope, thus, let keep this way.